pkg/scheduler/api/resource_info.go

```diff
@@ -87,7 +87,7 @@ func NewResource(rl v1.ResourceList) *Resource {
 			if v1helper.IsScalarResourceName(rName) {
 				ignore := false
 				for _, val := range IgnoredDevicesList {
-					if strings.Compare(val, rName.String()) == 0 {
+					if len(val) > 0 && strings.Contains(rName.String(), val) {
```

Comment:
Why do we have to change `compare` to `contains` for judgment? Are there any special considerations?

Reply:
fixed, I thought using strings.Contains will save me the trouble of trimming string.